src/NetworkExtension/NEAppProxyFlow.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+using System.Runtime.Versioning;
+using Foundation;
+using Network;
+using ObjCRuntime;
+
+namespace NetworkExtension {
+
+	public unsafe partial class NEAppProxyFlow {
+
+#if __MACCATALYST__ || MONOMAC
```

Comment:
`NEAppProxyFlow` is not available on tvOS, so this class needs conditional compilation as well.